### PR TITLE
EDURED-125: Login through QR from Home Screen PIN and Biometrics

### PR DESCRIPTION
**Pull request by risinggalaxy**

### CODE REVIEW

File: EduID/Flows/Scanning/VerifyScanResultViewController.swift [hunk 1]
```diff
@@ -111,10 +111,65 @@ class VerifyScanResultViewController: BaseViewController {
...
+        pinCodeVC.modalPresentationStyle = .fullScreen
+        present(pinCodeVC, animated: true)
+    }
+    
+    private func completeAuthentication(with mode: SecretVerificationMode, and pin: String? = nil) {
+        guard let challenge = viewModel.challenge as? AuthenticationChallenge,
+              let identity = challenge.identity else { return }
+        switch mode {
+        case .biometrics:
+            ServiceContainer.sharedInstance().secretService.secret(for: identity, touchIDPrompt: "Using Biometrics To Login") { [weak self] data in
```
Comment (dzolnai) [inline EduID/Flows/Scanning/VerifyScanResultViewController.swift:142]: Use localization files for strings

File: EduID/Utilities/Constants.swift [hunk 2]
```diff
@@ -52,10 +45,15 @@ enum Constants {
     enum KeyChain {
         static let keyPrefix = Bundle.main.bundleIdentifier ?? "nl.eduid"
         static let challenge = "Challenge"
+        static let oidAuthState = "EDU_ID_OIDAuthState"
```
Comment (dzolnai) [inline EduID/Utilities/Constants.swift:48]: I explicitly removed these, why are they back?

**Review by dzolnai — CHANGES_REQUESTED** [review]

File: EduID/Utilities/Constants.swift (hunk 2 above)
Comment (risinggalaxy) [reply EduID/Utilities/Constants.swift:48]: I must have used an old branch?!

Comment (dzolnai) [reply EduID/Utilities/Constants.swift:48]: A rebase should fix that then

**Review by dzolnai — APPROVED** [review]

**Pull request merged**